Library/Formula/tarantool.rb

```diff
@@ -0,0 +1,64 @@
+class Tarantool < Formula
+  desc "In-memory database and Lua application server."
+  homepage "http://tarantool.org"
+  url "http://tarantool.org/dist/master/tarantool-1.6.7-562-g3dbc9ff-src.tar.gz"
+  version "1.6.7-562"
+  sha256 "6aded257e11fa114dc21e8493ff96dfe3f8a692d62e096afb5105cecc179211d"
+
+  head "https://github.com/tarantool/tarantool.git", :shallow => false
+
+  devel do
+    url "https://github.com/tarantool/tarantool.git", :branch => "1.6", :shallow => false
+    version "1.6.8"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "readline" => :build
```

Comment:
Just `depends_on "readline"`. If it's available at runtime it'll be available at buildtime as well, essentially.